src/interface/src/app/plan/create-scenarios/create-scenarios.component.spec.ts

```diff
@@ -165,20 +210,20 @@ describe('CreateScenariosComponent', () => {
   //   ]);
   // });
 
-  it('creates uploaded project areas when event is emitted', () => {
-    component.scenarioConfigId = 1;
-    component.formGroups[0].get('priorities')?.setValue(['test']);
-    component.formGroups[2].get('uploadedArea')?.setValue(fakeGeoJson);
-    const router = fixture.debugElement.injector.get(Router);
-    spyOn(router, 'navigate');
+  // it('creates uploaded project areas when event is emitted', () => {
```

Comment:
Is this test no longer valid? Should we remove it instead of commenting it out?

Reply:
Sorry, should have added context in the pr description - as I'm making changes some of the tests are temporarily invalid, but will need to be added back in later. For example, we have temporarily disabled uploading project areas during scenario configuration so this test wouldn't pass and therefore would prevent the pr from being able to get merged (if you take a look at the GitHub actions, those are the angular and Django tests that automatically get run for each pr). 
You're right that it probably would be cleaner to remove it and organize the list of tests into a series of TODO's. I can do that in the next pr!